bio_hansel/scripts/get_sequence.py

```diff
@@ -0,0 +1,69 @@
+from typing import Dict
+
+import pandas as pd
+
+from Bio import SeqIO
+
+
+def get_sequences(output_directory: str, reference_genome_path: str,
```

Comment:
Can this function return the kmer sequences instead of writing to a file? We can have another function that writes to a file. 

Also this function should take the length of sequence to get around a SNP as a parameter.

Reply:
I separated the get_sequence and write_sequence functions into two separate functions in this commit f094037d9569b8290efc2136dc3d3bf58039c2e5